src/accel/Accel.ts

```diff
@@ -0,0 +1,138 @@
+import Chuck from "../Chuck";
+import { Accel_ck, AccelMsg_ck } from "./accelCk";
+
+
+/**
+ * Introducing Accel (accelerometer, on mobile) support for WebChucK. Accel wraps
+ * JavaScript DeviceMotionEvent listeners easing access to mobile device accelerometers
+ * in WebChucK code. 
+ *
+ * To get started with Accel:
+ * @example
+ * ```ts
+ * import { Chuck, Accel } from "webchuck";
+ *
+ * const theChuck = await Chuck.init([]);
+ * const accel = await Accel.init(theChuck); // Initialize Accel
+ * ```
+ * 
+ * The "devicemotion" event gives acceleration of the device on the three axes: x, y, and z. Acceleration is expressed in m/s².
+ * More on the "devicemotion" event can be found online {@link https://developer.mozilla.org/en-US/docs/Web/API/Window/devicemotion_event | here }.
+ * 
+ * iOS devices require that the web developer ask permission from the user to access sensors after a button push. For example: 
+ * 
+ * ```ts
+ * import { Chuck, Accel} from '../webchuck/src/wc-bundle.js';
```

Comment:
nit: import bracket spacing and can just make it `import { Chuck, Accel } from "webchuck"` like above